app/_hub/kong-inc/injection-protection/overview/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+nav_title: Overview
+title: Injection Protection
+---
+
+You can use the Injection Protection plugin to detect and block known injection patterns consistent with SQL injection, server-side include injection, and more.
+
+The Injection Protection plugin makes it easier to protect your APIs from SQL injection or other injection attacks by providing out-of-the box regex matching for common injection attacks. 
+You can also configure custom regex matching.
+
+The Injection Protection plugin does the following: 
+* Extracts information from request headers, path/query parameters, or the payload body and evaluates that content against predefined regular expressions
+* Rejects the requests that match the regular expressions with a configurable HTTP status code and error message
+* Logs information about rejected requests for analytics and reporting
+
+## How does the Injection Protection plugin work?
+
+Depending on what you have configured in the plugin's config, the Injection Protection plugin does the following:
```

Comment:
This line and line 11 say almost the same thing, but I think the difference is "what this plugin can do for you" (bad phrasing, but you get it) and "how this plugin literally works, in order of execution". Is that correct? If so, I'd rephrase to indicate that.